cpp/src/traversal/tsp.hpp

```diff
@@ -69,20 +80,43 @@ class TSP {
 
   // Scalars
   rmm::device_scalar<int> mylock_scalar_;
-  rmm::device_scalar<int> best_tour_scalar_;
+  rmm::device_scalar<int> best_cost_scalar_;
   rmm::device_scalar<int> climbs_scalar_;
 
   int *mylock_;
-  int *best_tour_;
-  int *climbs_;
+  int *best_cost_;
 
   // Vectors
   rmm::device_vector<int64_t> neighbors_vec_;
   rmm::device_vector<int> work_vec_;
+  rmm::device_vector<float *> best_x_pos_vec_;
+  rmm::device_vector<float *> best_y_pos_vec_;
```

Comment:
why `device_vector` is needed over over `uvector`?

Reply:
Updated here: https://github.com/rapidsai/cugraph/pull/1480/commits/beaf9fa68104fecd706bac2602bf31fff58e16a3